awx/conf/settings.py

```diff
@@ -406,6 +410,7 @@ def _get_default(self, name):
     def SETTINGS_MODULE(self):
         return self._get_default('SETTINGS_MODULE')
 
+    @cachetools.cached(cache=cachetools.TTLCache(maxsize=2048, ttl=SETTING_MEMORY_TTL))
```

Comment:
In my profiling for high workloads, the cache hits for redis actually add up and end up being quite expensive in bulk;   for the purposes of the callback receiver, it's probably not imperative that we have real-time correctness for these setting values (caching for a few seconds locally in memory makes a notable performance improvement).